### PR TITLE
OSDOCS-11819: Move bare metal config docs to bare metal install section

### PR DESCRIPTION
**Pull request by abrennan89**
Version(s):
4.17+

Issue:
https://issues.redhat.com/browse/OSDOCS-11819

Link to docs preview:
https://81172--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal_ipi/ipi-install-post-installation-configuration#bmo-config-using-bare-metal-operator_ipi-install-post-installation-configuration

QE review:
NA/ moving docs only, no change in content.

### CODE REVIEW

File: modules/bmo-config-using-bare-metal-operator.adoc [hunk 1]
```diff
@@ -1,12 +1,21 @@
...
+= Configuration using the Bare Metal Operator
 
-Use the Bare Metal Operator (BMO) to provision, manage, and inspect bare-metal hosts in your cluster.
+When deploying {product-title} on bare-metal hosts, there are times when you need to make changes to the host either before or after provisioning. This can include inspecting the host's hardware, firmware, and firmware details. It can also include formatting disks or changing modifiable firmware settings.
+
+You can use the Bare Metal Operator (BMO) to provision, manage, and inspect bare-metal hosts in your cluster. The BMO can complete the following operations:
+
+* Provision bare-metal hosts to the cluster with a specific image.
+* Turn on or off a host.
+* Inspects hardware details of the host and report them to the bare metal host.
```
Comment (lpettyjo) [inline modules/bmo-config-using-bare-metal-operator.adoc:15]: Change "Inspects" to "Inspect" for parallelism. Also, hyphenate "bare metal" before "host".

**Review by lpettyjo — COMMENTED** [review]
Please squash your two commits. Instructions here: https://docs.google.com/document/d/1fLMpK4bqthtFlCwA36yeo2cI-pExFpX76VMnSi_In6Q/edit#heading=h.epes4xf06pzk.

Otherwise, LGTM!

**Comment by abrennan89** [comment]
Cherrypicking to 4.17 only due to conflicts in previous release notes etc if links are updated

**Pull request merged**